helm/db-instances/templates/dbinstance.yaml

```diff
@@ -43,6 +43,11 @@ spec:
   {{- if $value.generic.publicIp }}
     publicIp: {{ $value.generic.publicIp }}
   {{- end }}
+  {{- if $value.generic.backupHost }}
```

Comment:
Would define this as a optional var. Use the default primary db from a fallback in the code.

(minor/optional)